src/process/__tests__/process.test.ts

```diff
@@ -3129,7 +3129,7 @@ describe('Process Tests', () => {
       processSync(context);
 
       expect(context.data).to.deep.equal({
-        candidates:[{candidate:{data:{section6:{}}}}],
+        candidates:[{candidate:{data:{section6:{ "c":{}, "d":[]}}}}],
```

Comment:
This test appeared to be doing invalid assertion, since `clearOnHide` is false in this case, the fields should be left. Corrected it.